app/helpers/ems_cloud_helper/textual_summary.rb

```diff
@@ -146,7 +146,8 @@ def textual_security_groups
   end
 
   def textual_zone
-    {:label => _("Managed by Zone"), :icon => "pficon pficon-zone", :value => @record.zone.name}
+    zone = @record.zone.visible? ? @record.zone : @record.backup_zone
```

Comment:
1. This method is not called backup_zone anymore...I think it's zone_before_pause

2. I feel like this section is wrong.  This is showing the EMS as being in a zone that it's not really in, which can be confusing to end users as well as us trying to diagnose customer issues.  IMO, we should just keep the original code and show that it's in the Maintenance Zone or wherever.  Or, clarify it better with something like the following...

```ruby
zone = @record.zone_before_pause ? "#{@record.zone.name} (originally in #{@record.zone_before_pause.name})" : @record.zone.name
```

Reply:
Same in the copy-paste areas below.